src/H5CXprivate.h

```diff
@@ -115,6 +115,9 @@ H5_DLL herr_t H5CX_get_filter_cb(H5Z_cb_t *filter_cb);
 H5_DLL herr_t H5CX_get_data_transform(H5Z_data_xform_t **data_transform);
 H5_DLL herr_t H5CX_get_vlen_alloc_info(H5T_vlen_alloc_info_t *vl_alloc_info);
 H5_DLL herr_t H5CX_get_dt_conv_cb(H5T_conv_cb_t *cb_struct);
+H5_DLL herr_t H5CX_get_selection_io_mode(H5D_selection_io_mode_t *selection_io_mode);
+H5_DLL herr_t H5CX_get_no_selection_io_cause(uint32_t *no_selection_io_cause);
+H5_DLL herr_t H5CX_get_modify_write_buf(hbool_t *modify_write_buf);
```

Comment:
No need for set routines for selection io mode or modify write buf?

Reply:
Not needed by the library since the library never changes it. There are several other properties where this is the case. For something like this it's best not to create this routine since we really want to make sure we don't violate const if the app didn't set this property.